presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -438,6 +442,11 @@ private ConnectorTableMetadata getTableMetadata(SchemaTableName tableName)
 
         Optional<String> comment = Optional.ofNullable(table.get().getParameters().get(TABLE_COMMENT));
 
+        String collectColumnStatisticsOnWrite = table.get().getParameters().get(COLLECT_COLUMN_STATISTICS_ON_WRITE_ENABLED_KEY);
+        if (collectColumnStatisticsOnWrite != null && Boolean.valueOf(collectColumnStatisticsOnWrite)) {
```

Comment:
I don't like `Boolean.valueOf` because it accepts `"yes"` but returns `false`

Reply:
Removed